### ISSUE TITLE
Adding support for GFM with prism syntax highlighting.

### ISSUE THREAD

**Issue by Blackbaud-AaronDershem** [blackbaud-community/stache#33]
To avoid having to use the `<pre><code>` tags as mentioned in the [documentation](https://github.com/blackbaud-community/stache-docs/blob/master/content/docs/css-js/index.md#syntax-highlighting):

```
<pre><code class="language-js">
var a='hello';
console.log(a + 'world');
</code></pre>
```

and instead be able to use Github Flavored Markdown with backticks and syntax highlighting with prism.js:

``````
    ``` js
    var a='hello';
    console.log(a + 'world');
    ```
``````

Prism requires a prefix of exactly "language-" and blackbaud-marked was only using the default of "lang-".  I made the following change in [helper.js](https://github.com/blackbaud/stache/blob/master/src/helpers/helpers.js#L138).  

``` js
// /src/helpers/helpers.js
    function getMarked(md) {

        var comment = '\n<!-- STACHE MARKED -->\n',
            input = md || '';

        return input.indexOf(comment) > -1 ? input : [
            comment,
            marked.parser(lexer.lex(input), {
                headerPrefix: '',
                renderer: renderer,
               /*
               // Changes prefix to be "language-" as required by prism.js
               */
                langPrefix: 'language-' 
            })
        ].join('');
    }
```

Is this an acceptable solution?  If it is can we have this added to the master branch?  If not we can create a fork.

**Comment by Blackbaud-BobbyEarl** [comment]
I would be happy to accept that pull request (or just add it).  We'd previously advised against using the three backticks method, as our render would process nested content incorrectly, so we got a lot of inconsistent behavior.

In the past few months, we resolved that issue, which is what the '' comment is there for, but we never revisited the backticks.

Also, in both your examples you reference `js`, but I believe prism only honors `javascript`.  Your suggestion is still valid though.

Great catch!
